packages/plugins/rubygem-foreman_webhooks/rubygem-foreman_webhooks.spec

```diff
@@ -34,6 +34,7 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
 BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 4.15.1
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 5.0.0
```

Comment:
DEBUG util.py:598:  Error: No Package found for tfm-npm(@theforeman/builder) < 5.0.0

Please drop

Reply:
Thanks, dropped.